packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -455,7 +473,13 @@ export const CreatePlanFormContainer = ({
     }
   }, [technicalList]);
 
-  const { mutate: onCreatePlan } = useSavingsPlanCreate();
+  const { mutate: onCreatePlan } = useSavingsPlanCreate(async (data) => {
+    addSuccess(
+      tListing('banner_create_sp', {
+        startDate: toLocalDateUTC(data.startDate, locale),
+      }),
+    );
+  });
```

Comment:
I think this should be wrapped in a useCallback to avoid unnecessary re-render

Reply:
And please, make a dedicated function to prevent from adding an anonymous function as hook parameter 👍🏻